R/commonMachineLearningClassification.R

```diff
@@ -136,7 +148,7 @@
   }
 
 # If no analysis is run, specify the required variables in a footnote
-  requiredVars <- ifelse(type == "knn", yes = 1L, no = 2L)
+  requiredVars <- if (type == "knn" || type == "neuralnet") yes = 1L else no = 2L
```

Comment:
```suggestion
  requiredVars <- if (type == "knn" || type == "neuralnet") 1L else 2L
```
otherwise, you'll create a variable `yes` (or `no`) depending on whether the condition is true or not.